react/components/SearchBar/components/ResultsList.js

```diff
@@ -9,10 +10,17 @@ import autocomplete from '../queries/autocomplete.gql'
 
 import styles from '../styles.css'
 
+const WrappedSpinner = () => (
+  <div className="w-100 flex justify-center">
```

Comment:
maybe drop w-100 and use flex-grow-1 ?
@lbebber what do you think?

Reply:
sounds good if the parent is a flex element

Reply:
It's not